conductor/src/types.rs

```diff
@@ -33,6 +33,8 @@ pub enum Event {
     Started,
     Restore,
     Restored,
+    ScheduleDeletion,
+    ScheduleDeletionComplete,
```

Comment:
Make sure these align exactly with what the control-plane is expecting when serialized.